CHANGELOG.md

```diff
@@ -7,7 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
```

Comment:
just in case, this should be version 4.x.x since we're changing requirements.

Reply:
@dmitry-mukhin surely